DigitalExcellence/dex-frontend
# [Bug] {#556] Adding maxlength validator and error message for Project Name

## Description

Adding maxlength validation to Project Name step of Add Project Wizard.  Checking project name length at step 1 of wizard vs after step 8's API call.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to step 1 of the Add Project wizard...
2. Add over 75 characters to the name field...
3. After that, you should see the error message and the Next button disabled.

Project Name Initial State
![project-name-initial-state](https://user-images.githubusercontent.com/21061492/137505548-087c9cb7-b370-4418-a760-edcf6f057244.PNG)

Project Name Error Message
![error-message-closer](https://user-images.githubusercontent.com/21061492/137505731-42a551cb-68f4-4d5e-b2fd-ae3c3317fad1.PNG)

Project Name Valid Name
![project-name-valid-message](https://user-images.githubusercontent.com/21061492/137505550-83f8ca56-1600-498d-a106-e605c851874e.PNG)

## Link to issue


Closes: #556
